proto/connectrpc/conformance/v1alpha1/client_compat.proto

```diff
@@ -40,30 +40,20 @@ message ClientCompatRequest {
   string service = 9;
   string method = 10;
   StreamType stream_type = 11;
-  repeated Header request_headers = 12;
+  // If protocol indicates Connect and stream type indicates
+  // Unary, this instructs the client to use a GET HTTP method
+  // when making the request.
+  bool use_get_http_method = 12;
+  repeated Header request_headers = 13;
   // There will be exactly one for unary and server-stream methods.
   // For client- and bidi-stream methods, all entries will have the
   // same type URL (which matches the request type of the method).
-  repeated google.protobuf.Any request_messages = 13;
-  optional uint32 timeout_ms = 14;
+  repeated google.protobuf.Any request_messages = 14;
+  optional uint32 timeout_ms = 15;
   // Wait this many milliseconds before sending a request message.
   // For client- or bidi-streaming requests, this delay should be
-  // applied before each request sent
-  uint32 request_delay_ms = 15;
-  // Tells the server whether it should wait for each request
-  // before sending a response.
-  // If true, it indicates the server should effectively interleave the
-  // stream so messages are sent in request->response pairs.
-  // If false, then the response stream will be sent once all request messages
-  // are finished sending with the only delays between messages
-  // being the optional fixed milliseconds defined in the response
-  // definition.
-  // This field is only relevant in the first message in the stream
-  // and should be ignored in subsequent messages.
-  // Note, this is only applicable to bidi endpoints.
-  // TODO - This could potentially be removed bc this is in the BidiStreamRequest
-  // which would be in the request_messages field
```

Comment:
This is now implied by the stream type field. I suppose I might preserve the TODO that this is technically still in two places -- in this message and in the response definition of the request.

Reply:
Yeah I'd agree with that. I can see having the two fields, but we could at least have a comment for how they interact and why they both exist.

Reply:
We could even make the code that loads and processes test suite configs auto-populate the "full_duplex" field in the test case response definition based on this stream type, which would eliminate the redundancy.